packages/panda-san/src/navbar/navbar.scss

```diff
@@ -0,0 +1,82 @@
+.navbar {
+  align-items: center;
+  background-color: $navbar-color;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  left: 0;
+  position: relative;
```

Comment:
I don't think we need this unless there is an absolutely positioned element within the `navbar` which I don't see? The `left: 0` style can be removed in that case as well.